### PR TITLE
Graph : Move controlFlowGraph into it's own class to allow for stuff …

### PR DESCRIPTION
**Pull request by Herz3h**
…to properly be deleted when reloading the program/loading a different program

Graph : Fixed two bugs causing crashes + some cleanup/refactor

### CODE REVIEW

File: src/dbg/_global.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #ifndef _GLOBAL_H
 #define _GLOBAL_H
 
-#define _WIN32_WINNT 0x0501
```
Comment (mrexodia) [inline src/dbg/_global.h]: yea this, don't do it. we need xp support

Comment (Herz3h) [inline src/dbg/_global.h]: Well GetProcessDepPolicy requires it to be 0x600

**Comment by Nukem9** [comment]
@Herz3h 
https://github.com/x64dbg/x64dbg/commit/5582730485f9851ce5a1f2436f3edd034a46468e

aka the hacky-but-compatible-way

**Comment by mrexodia** [comment]
Yea we talked about it in Gitter, come join is 😀

**Pull request merged**